backend/db/schema.rb

```diff
@@ -91,6 +91,7 @@
     t.string "box_office"
     t.string "rating"
     t.string "order"
+    t.string "trailer"
```

Comment:
This won't work as the schema will be regenerated every time you run `rails db:migrate`. Therefore you should probably create a migration to add the trailer field to the movies column. 
You also should add the field to the [movie serializer](https://github.com/danielschuster-muc/potter-db/blob/master/backend/app/serializers/movie_serializer.rb) (else you wouldn't be able to get in the api response), as well as in the [seeds file](https://github.com/danielschuster-muc/potter-db/blob/master/backend/db/seeds.rb#L74).
And for the sake of completeness also in the [movies factory](https://github.com/danielschuster-muc/potter-db/blob/master/backend/spec/factories/movies.rb).

Reply:
If you don't have any experience with Ruby on Rails, this would be totally fine, too.
I can also work on the part of implementing it in the backend. Just tell me what you prefer.